.devcontainer/vmr-source-build/README.md

```diff
@@ -47,11 +48,8 @@ Let's consider you pushed a commit with SHA `abcdef` into your fork at `github.c
 You can now bring this version of runtime into the local VMR in this Codespace by running:
 
 ```bash
-cd /workspaces/installer
-./eng/vmr-sync.sh             \
-  --vmr /workspaces/dotnet    \
-  --tmp /workspaces/tmp       \
-  --repository runtime:abcdef \
+/workspaces/synchronize-vmr.sh \
+  --repository runtime:abcdef  \
```

Comment:
It's pretty easy to figure out the argument format from this example, especially for the repo part, but I'd suggest to use standard notation, such as `--repository <repo>:<commit>`.

Reply:
I made these more generic